web/libs/datamanager/src/components/Common/Table/Table.scss

```
@@ -0,0 +1,155 @@
+.column-selector {
+  top: 0;
+  right: 0;
+  width: 100%;
+  display: flex;
+  overflow: hidden;
+  position: absolute;
+  justify-content: flex-end;
+
+  &__button {
+    padding: 0;
+    z-index: 1000;
+    border-radius: 0;
+    height: 44px;
+    width: 45px;
+    margin: -1px;
+    box-shadow: none;
+    border-left: 1px solid rgba(#000, 0.1);
+  }
+
+  &__button-new&__button-new {
+    z-index: 1000;
+    height: 28px;
+    width: 28px;
+    background-color: #fff;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
+    margin: 6.5px 20px;
+    border-radius: 4px;
+
+    &:hover {
+      border: none;
+      filter: none;
+      background: #ebf3fc;
+    }
+
+    &:active {
+      background: linear-gradient(0deg, rgba(#000, 0.02), rgba(#000, 0.02)), #FFFFFF;
+      box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.2);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
      background: linear-gradient(0deg, var(--black_2), var(--black_2)), #FFFFFF;
      box-shadow: 0 0 5px 0 var(--black_20);
```